From 496465d9238109a93612e28682273e5bf576823b Mon Sep 17 00:00:00 2001
From: Chen Qi <Qi.Chen@windriver.com>
Date: Sun, 22 May 2022 19:40:59 -0700
Subject: [PATCH 1/2] buffer.h: add missing header file due to gcc upgrade

The header file <memory> have been changed to be no longer included by other
heades, thus requiring explicit include. See https://gcc.gnu.org/gcc-12/porting_to.html

Upstream-Status: Pending

Signed-off-by: Chen Qi <Qi.Chen@windriver.com>
---
 src/include/buffer.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/include/buffer.h b/src/include/buffer.h
index 5c8b427d..88845ee6 100644
--- a/src/include/buffer.h
+++ b/src/include/buffer.h
@@ -43,6 +43,7 @@
 #include <list>
 #include <vector>
 #include <string>
+#include <memory>
 #if __cplusplus >= 201703L
 #include <string_view>
 #endif // __cplusplus >= 201703L
-- 
2.36.0

